### DISCUSSION TITLE
İBAX Rank

### DISCUSSION THREAD

**Discussion by coinsinov** [IBAX-io/go-ibax#695] [Soru Cevap - Türkçe]
Reetwet tamamladı, puan alma yeri kalmadı. İBAX Rank atlamak için başka neler yapmalıyız? Saygılar.

**Comment by Yusufknk** [comment]
Görevler ve çekilişler için sosyal medya kanallarını takipte kalın.

**Comment by hsynkader** [comment]
> Reetwet tamamladı, puan alma yeri kalmadı. İBAX Rank atlamak için başka neler yapmalıyız? Saygılar.

Sizin kaç puanınız var toplamda.

**Reply by Yusufknk** [reply]
22 bin

**Reply by Garcanlar28** [reply]
nasilnpuan alcam github dan

**Reply by Yusufknk** [reply]
İbax Türkiye telegram grubunda sabitlenmiş mesajlarda nasıl yapılacağı mevcut.

**Comment by coinsinov** [comment]
21 Bin var benimde

**Comment by hsynkader** [comment]
Benimde durum böyle
Infantry
IBAX Points: 22,885

**Comment by Garcanlar28** [comment]
nasol github dan puan alcam benim 0

**Comment by Garadepelim** [comment]
yeni retweet geldi
